packages/mobile/src/screens/search-results-screen/tabs/AlbumsTab.tsx

```diff
@@ -1,21 +1,27 @@
-import { SearchKind, searchResultsPageSelectors } from '@audius/common'
-import { useSelector } from 'react-redux'
+import type { CommonState } from '@audius/common'
+import { useProxySelector, SearchKind } from '@audius/common'
 
 import { CollectionList } from 'app/components/collection-list/CollectionList'
 
 import { SearchResultsTab } from './SearchResultsTab'
 import { useFetchTabResultsEffect } from './useFetchTabResultsEffect'
 
-const { makeGetSearchAlbums } = searchResultsPageSelectors
-
-const getSearchAlbums = makeGetSearchAlbums()
+const selectSearchAlbums = (state: CommonState) =>
+  state.pages.searchResults.albumIds
+    ?.map((albumId) => {
```

Comment:
for all these tabs might be worth to share this mapping/filtering logic w the `makeGetSearchX` functions in common (since they seem to be the same). and maybe even move this to a separate selectors file (either the one in common or a new one in mobile since this is mobile only)

Reply:
yeah we should really replace the existing selectors in common with these, just wanted to go quickly here and make sure im not messing up desktop. added a tech-debt item to move these over to common tho!